articles/nginx-reverse-proxy/index.md

```diff
@@ -0,0 +1,161 @@
+# Deploying Multiple Applications to VM with NGINX as a Reverse Proxy
+
+## Introduction
+Ever wondered how more than one application is deployed to the same machine, and how traffic is routed to the corresponding applications?
+Keep reading to find out.
+## Goals
+By the end of the article, you'll understand
+- What a Reverse Proxy is?
+- What NGINX is?
+- How does NGINX help in managing multiple applications?
+- How to leverage NGINX as a Reverse Proxy?
+
+### Reverse Proxy
+A reverse proxy is a type of proxy server that retrieves resources on behalf of a client from one or more servers. These resources are then returned to the client, appearing as if they originated from the server  itself.  
+ Refer [this article](https://www.section.io/engineering-education/what-are-reverse-proxies/) to better understand what Reverse Proxies are.
+
+### NGINX
+We will be using NGINX which would be serving the purpose of a Reverse Proxy. NGINX is a web server that can be used as a reverse proxy, load balancer, mail proxy and HTTP cache. The software was created by Igor Sysoev and publicly released in 2004. Nginx is free and open-source software, released under the terms of the 2-clause BSD license. A large fraction of web servers use NGINX, often as a load balancer.
```

Comment:
We will be using NGINX **as a reverse proxy** would be more succinct.

Reply:
Yeah this is better. Thanks